### ISSUE TITLE
Cannot press F-PLN Button

### ISSUE THREAD

**Issue by pezza72** [flybywiresim/a32nx#4637]
**Aircraft Version**

```json
{
  "built": "2021-04-15T22:57:32+00:00",
  "ref": "v0.6.0",
  "sha": "981b2e4b0e89998d21a70fe26de54a2f10066f6e",
  "actor": "Benjozork",
  "event_name": "manual"
}
```
Tested with both Stable and Development versions using the installer.

**Describe the bug**

Clicking on the "F-PLN" button makes the clicking sound but doesn't appear to do anything else.  It doesn't display the Flightplan Screen.  I have checked on the standard A320neo and it works with that.

**To Reproduce**

Example flight data used:
From: Doha International (OTBD) (RW33 ALVE1N)
To: Dubai International (OMDB) (RW12R DATO1E)
IFR-Low Altitude

1. Load flight (tried with the following:)
a) Plan selected with MSFS ONLY and NO simBrief
b) As per a) but linking to simBrief in the MCDU
c) Flight Plan loaded from simBrief at flight start up

2. Enter INIT and PERF data
3. Click on F-PLN (also tried selecting F-PLN before entering INIT and PERF)

**Expected behavior**

Flight Plan page loads showing the plan

**Actual behavior**

Flight Plan page does not show
Button click sound works

**References**

There is a bug filed for this same issue which has a video attached that shows the same symptoms.  Unfortunately the same action of re-installing didn't work for me.

**Additional context**

I have tried removing the install from the community folder completely and then re-installing with the installer but I get the same issue.

Was this working before/when did the issue start occurring?
Yes.  Seems to have occurred since the update to 0.6 and/or the MSFS Update.  I updated through the installer first to 0.6 and then the update kicked in in MSFS upon launch, so only able to test when BOTH were updated.

Is this a problem in the vanilla unmodded game?
Tested with the a320neo and the button works fine and displays the flight plan

**Comment by MisterChocker** [comment]
Do you have anything else in your community folder besides the a32nx?
If so there might be a mod conflict. I suggest temporarily removing all other addons/mods, retest and if the issue no longer persists re-adding one addon at a time to find the trouble maker.

**Comment by pezza72** [comment]
Hi,

Thanks for the reply, and this was the issue.

Problem solved.

I only had navigraph data and maintenance in the community folder other than this mod, so I removed both Navigraph folders and this did resolve the issue.

Adding back just the Navigraph data re-introduced the problem, and that was also the case with the maintenance as well (which I expected given this was the original scenario)

Having checked a couple of posts, I think my issue was that I was running on an old 2013 cycle.  Still a bit unsure exactly as in MSFS Content Manager I had 3 "Not Installed" which show as the A32NX, Navigraph 2013 and Navigraph Maintenance but no option to install. 

The baffling thing is I don't know when and what actually changed as it seems that I was missing the Navigraph .exe and I don't recall ever downloading it before, yet this had always worked.  I can only assume that it was a combination of the recent updates, probably mostly MSFS

Anyway long story...still long!  I installed navigraph.exe which automatically showed 2013 cycle installed.  Tried again but still couldn't press the F-PLN button.  I then followed the instructions here [(https://forum.navigraph.com/t/faq-how-to-update-the-airac-files-new-cycle-or-new-revision/4136)](https://forum.navigraph.com/t/faq-how-to-update-the-airac-files-new-cycle-or-new-revision/4136) to update the Cycle and it then showed 2014.

Whilst it still shows the AIRAC Cycle as Not Installed in MSFS, it does now show 2014 and the "Navigraph Maintenance" no longer shows as Not Installed (not changed anything with that unless it was done internally).

And the most important part is that after all that, I can now hit the F-PLN button and I have the plan back on screen.

So it just all seems related to the old AIRAC Cycle causing conflicts.

Sorry for long comment, but hopefully this may help others.
